less/common/mixins/header-background.less

```
@@ -6,7 +6,7 @@
   right: 0;
   z-index: @zindex-header;
   border-bottom: 1px solid @control-bg;
-  .transition(~"box-shadow 0.2s, -webkit-transform 0.2s");
+  transition: ~"box-shadow 0.2s, -webkit-transform 0.2s";
```

Not sure what the `-webkit-transform` here is for, there isn't even a transform property 🤔 but either way we can change that to `transform`.

```suggestion
  transition: box-shadow 0.2s, transform 0.2s;
```